contracts/SplitWallet.sol

```diff
@@ -59,12 +66,18 @@ contract SplitWallet {
 
   constructor() {
     splitMain = ISplitMain(msg.sender);
+    SPLIT_DEPLOYER = tx.origin;
```

Comment:
would maybe call this `SPLITS_DEPLOYER` so as to not confuse with an individual split deployer